kratos/tests/cpp_tests/geometries/test_quadrature_point_geometry.cpp

```diff
@@ -104,5 +133,44 @@ namespace Kratos {
             KRATOS_CHECK_EQUAL(p_this_quadrature_point->ShapeFunctionsValues().size2(), 3);
         }
 
+        KRATOS_TEST_CASE_IN_SUITE(QuadraturePointGeometry2dCopyConstructor, KratosCoreFastSuite)
+        {
+            auto p_this_quadrature_point = GenerateQuadraturePointGeometry2();
+
+            auto geom = QuadraturePointGeometry<Node<3>, 2, 2>(*p_this_quadrature_point);
+
+            KRATOS_CHECK_EQUAL(geom.size(), 3);
```

Comment:
why not like this?
```suggestion
            KRATOS_CHECK_EQUAL(geom.size(), p_this_quadrature_point->size());
```

otherwise if for whatever reason you change `GenerateQuadraturePointGeometry2` you have to completely change this test

Reply:
I used to make that to avoid that we are still pointing to the original geometry and might have changed something there... I think it does not really add additional safety, here.

Reply:
I used to do this because then you're safe that you are not just pointing to the original geometry. However, in this case you're right, it does not really improve the safety...

Reply:
ok